### DISCUSSION TITLE
initWasm getting error

### DISCUSSION THREAD

**Discussion by wanderlust252** [trustwallet/wallet-core#3193] [Q&A]
**Describe the bug**
![image](https://github.com/trustwallet/wallet-core/assets/34784609/6692e8b0-233f-45fd-816a-d3a4357b7166)

**To Reproduce**
Im creating Chrome Extension with Vite + Svelte + UnoCSS
My code copy from Sample. I don't know why it getting error

**Screenshots**
![image](https://github.com/trustwallet/wallet-core/assets/34784609/e6ee6120-491c-42f0-a198-852744491568)

**Comment by wanderlust252** [comment]
I tried many ways but seem like this core hasn't supported browser extension yet.

**Reply by fukemy** [reply]
hi, do you found any solution? I got same issues with NextJS

**Comment by weixuefeng** [comment]
1. build your own wallet-core.wasm & wallet-core.js [link](https://github.com/trustwallet/wallet-core/issues/3234)
2. copy build result to your project, maybe you should rewrite [fs-stroage](https://github.com/trustwallet/wallet-core/blob/master/wasm/src/keystore/fs-storage.ts) file.

**Comment by weixuefeng** [comment]
<img width="1215" alt="image" src="https://github.com/trustwallet/wallet-core/assets/25627103/f00a4069-9689-4966-bcea-6877cf552b4f">

**Reply by fukemy** [reply]
@weixuefeng Can I copy the wasm file from node_modules to root project and use it?

**Reply by weixuefeng** [reply]
yes, you have to do it.

**Reply by fukemy** [reply]
Yeah, I used the `wallet-core.wasm` from node-modules, then I got this error: 
```
2153272 - Exception catching is disabled, this exception cannot be caught. Compile with -sNO_DISABLE_EXCEPTION_CATCHING or -sEXCEPTION_CATCHING_ALLOWED=[..] to catch.
```
Can you please help my problem
